### DISCUSSION TITLE
How can I contact repo admin?

### DISCUSSION THREAD

**Discussion by meremeev** [open-mmlab/mmdetection3d#2025] [Q&A]
I need to completely remove some PR, which was submitted by mistake.

**Comment by ZCMax** [comment]
Sorry, Can you provide more details?

**Comment by meremeev** [comment]
Sure, 
By mistake I have pushed some proprietary code in `open-mmlab` repo instead of our fork. I have closed PR, but it is still visible. I would like to completely remove it from `open-mmlab` repo.
PR to remove is https://github.com/open-mmlab/mmdetection3d/pull/1675

Thanks.
